packages/app/app/studio/[organization]/library/components/UploadVideoDialog.tsx

```diff
@@ -0,0 +1,84 @@
+'use client'
+
+import { Separator } from '@/components/ui/separator'
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@/components/ui/dialog'
+import { Button } from '@/components/ui/button'
+import UploadVideoForm from './upload/UploadVideoForm'
+import { useState } from 'react'
+import { Video } from 'lucide-react'
+
+const UploadVideoDialog = ({
+  organizationId,
+  organizationSlug,
+}: {
+  organizationId: string
+  organizationSlug: string
+}) => {
+  const [isUploaded, setIsUploaded] = useState(false)
+
+  const onFinish = () => {
+    setIsUploaded(true)
+  }
+
+  const reload = () => {
+    location.reload()
+  }
+
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button variant={'primary'} className="text-white">
+          Upload video
+        </Button>
+      </DialogTrigger>
+      <DialogContent className="sm:max-w-[525px] sm:max-h-[800px]">
+        {isUploaded ? (
+          <>
+            <DialogHeader className="p-10 space-y-4">
+              <div className="p-4 mx-auto bg-purple-500 rounded-full border">
```

Comment:
What hex is the new StreamETH Purple?

Reply:
`bg-primary` while some buttons have linear gradients applied to them. Instead you can use the variant="primary"

`primary:
          'bg-gradient-to-b from-[#3D22BA] to-[#6426EF] border-primary',`